tools/make-pkgs/readme.md

```diff
@@ -0,0 +1,36 @@
+## make-pkgs
+**make-pkgs** is a collection of bash scripts, intended to make the experience of compiling OBK and generating packages for various distros simpler and user-friendly.
+This is made as an user-facing tool, with simple switches for operation.
+\
+**NOTE** : It is **HIGHLY** recommended you have the `toolbox` package installed.
+
+## The importance of Toolbx containers
+Toolbx is a user-friendly CLI utility for creating and entering "containers" for various linux distros, similar to Distrobox.
+\
+Toolbx is used in make-pkgs to compile and package OBK for distros other than your own.
+\
+For example, If you are on Debian and you want to generate `.rpm`s specifically for Fedora 37, you can do so.
+
+## How to use
+Copy this folder, make the make-pkgs.sh script executable by doing `chmod +x make-pkgs.sh`
```

Comment:
@larina3315 why do we need to copy this folder?

Reply:
I was writing this from the POV that if a user clones the OBK git repo with this tool in it, then the make-pkgs.sh file will reside in ./OpenBangla-Keyboard/tools/make-pkgs/ , so the user might want to copy the folder to some other conveinent location and run it from there.

It could be worded in a better way as a suggestion.